src/picongpu/include/particles/gasProfiles/SphereFlanksImpl.hpp

```diff
@@ -66,20 +66,20 @@ struct SphereFlanksImpl : public T_ParamClass
 
         if (globalCellPos.y() < vacuum_y) return float_X(0.0);
 
-        const float_X radius = math::abs(globalCellPos - center);
+        const float_X distance = math::abs(globalCellPos - center);
```

Comment:
This was the only one variable in the gas implementation which I renamed becaus eI was confused that we had `radius`, `r` and `ri`